config/config.go

```diff
@@ -59,15 +58,15 @@ func (c *Config) ParseEnv() {
 	var failedParameters []EnvParameter
 	// loop through registered parameters and try to find them in env
 	for k, parameter := range config.Environment {
-		envValue, found := syscall.Getenv(parameter.EnvName)
+		envValue := os.Getenv(parameter.EnvName)
 		// if mandatory but not found add them to error list
-		if found == false && parameter.Mandatory == true {
+		if envValue == "" && parameter.Mandatory == true {
```

Comment:
we can not test env value for empty string because this might be a valid value, e.g. empty passwords. Please use os.LookupEnv() above on Line 61.

Reply:
Fixed.